.github/workflows/core_build_publish.yml

```diff
@@ -97,7 +105,12 @@ jobs:
       - name: Test
         run: dotnet msbuild /t:RunTestsWithCoverage
         working-directory: test
-
+      - name: Integration Test
```

Comment:
TBH I would move integration tests to a separate workflow. It will be easier to manage/see the problem there.

Reply:
Can we orchestrate workflows so that publish job is done only if integrations tests pass?